package.json

```
@@ -1,10 +1,16 @@
 {
   "scripts": {
     "test": "jest --coverage",
+    "ui-test": "jest test/js --testPathIgnorePatterns=scp-api.test.js",
```

Blocker: my understanding is the intent here is to ignore `test/js/fetch/scp-api.test.js`, but it also ignores `test/js/scp-api.test.js` -- which seems like a functional bug insofar as CI is concerned.

Something like the following would likely test and skip as desired.

```suggestion
    "ui-test": "jest test/js --testPathIgnorePatterns=fetch/**",
```

---

Made updates in 0e230023. 
Results after update to use ui-test:
```Test Suites: 14 passed, 14 total
Tests:       40 passed, 40 total
Snapshots:   0 total
Time:        4.678s, estimated 6s
Ran all test suites matching /test\/js/i.
 ```
Results using yarn test: 
```
Test Suites: 15 passed, 15 total
Tests:       42 passed, 42 total
Snapshots:   0 total
Time:        5.55s
Ran all test suites.
✨  Done in 7.00s.
```
UI-test excludes 2 test so these results are expected